deployment/v3/utils/purge-identity-cache/install.sh

```
@@ -0,0 +1,87 @@
+
+# ====== CONFIGURATION ======
+NAMESPACE="clear-identity-caches"
+CRONJOB_NAME="clear-identity-cache"
+SECRET_NAME="db-credentials"
+DB_NAME="mosip_ida"
+DB_USER="postgres"
+
+CONFIGMAP_NAME="db-config"
+
+# ====== USER INPUT ======
+read -p "Enter DB Host: " DB_HOST
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)